enterprise/server/remote_execution/containers/podman/BUILD

```diff
@@ -82,6 +82,11 @@ genrule(
         # Remove README.md (not needed)
         rm "$$TMPDIR/README.md"
 
+        # Remove container runtimes that are included in podman-static
+        # release, since we provision the container runtime separately.
+        rm "$$TMPDIR/usr/local/bin/crun"
+        rm "$$TMPDIR/usr/local/bin/runc"
```

Comment:
do you think it'd make sense to run a command that verifies there's no crun/runc after these are removed and fails if not, just in case an upstream change somehow reintroduces them?

Reply:
These `rm` commands will fail if the `crun` / `runc` binaries didn't actually exist at these paths. I think that should be good enough - wdyt?

Reply:
sgtm